x-pack/plugins/ml/public/application/components/navigation_menu/main_tabs.tsx

```diff
@@ -25,6 +24,10 @@ interface Props {
   tabId: TabId;
 }
 
+const ML_APP_NAME = i18n.translate('xpack.ml.navMenu.mlAppNameText', {
```

Comment:
should be placed in the constants folder

Reply:
Moved the constants here https://github.com/elastic/kibana/pull/76735/commits/68e75199438fe6ce84017c80903ad2eaa43cfdc7